src/main.rs

```diff
@@ -37,6 +37,15 @@ pub enum Command {
     Plan,
     /// Replaces the library with exactly what is in the lock file
     Sync,
+    /// Add to a simple package to the project and sync
+    Add {
+        #[clap(value_parser)]
+        packages: Vec<String>,
+        #[clap(long)]
+        plan: bool,
+        #[clap(long)]
+        no_sync: bool,
```

Comment:
what's the difference between plan and no-sync? Can you add `///` comments above them so they get help text?